client/components/pages/Splash.jsx

```diff
@@ -1,26 +1,50 @@
-import React from 'react'
+import React, {PureComponent} from 'react'
+
+import SignUp from '../parts/SignUp'
```

Comment:
We shouldn't be using relative paths in case we refactor and components get moved around. It's best to set module directories in the web pack config.
Currently the components directory is set, so you could use: 'parts/SignUp' -- do others think we should add parts as a module directory or what I just wrote okay?